src/main/java/com/flowingcode/vaadin/addons/demo/TabbedDemo.java

```diff
@@ -206,24 +206,37 @@ public void showRouterLayoutContent(HasElement content) {
   public void removeRouterLayoutContent(HasElement oldContent) {
     getElement().removeChild(1);
   }
+  
+  private void updateSplitterPosition() {
+    setSourceVisible(codeCB.getValue());
+  }
 
   public void setSourceVisible(boolean visible) {
-    sourceVisible = visible;
-    if (currentLayout != null) {
-      currentLayout.setSplitterPosition(visible ? 50 : 100);
-      orientationCB.setEnabled(visible);
+    currentLayout.setSplitterPosition(visible ? 50 : 100);
```

Comment:
It should be possible to call `setSourceVisible` when there is no currentLayout (it just does nothing; the demo site relies on that behavior)